website/pages/en/_meta.js

```diff
@@ -7,7 +7,6 @@ export default {
   network: 'The Graph Network',
   sunrise: 'Sunrise FAQ',
   billing: '',
-  'network-transition-faq': '',
```

Comment:
this hidden route produced offset - 1 issue since, I used `flatDirectories` list instead of `flatDocsDirectories`

because in original `nextra-theme-docs` were props naming confusing

https://github.com/shuding/nextra/blob/2f001d897f2109b24600f66fe2009e5f788b980c/packages/nextra-theme-docs/src/components/nav-links.tsx#L22
https://github.com/shuding/nextra/blob/2f001d897f2109b24600f66fe2009e5f788b980c/packages/nextra-theme-docs/src/index.tsx#L220

Reply:
Really nice catch!